tests/conftest.py

```diff
@@ -56,3 +64,45 @@ def init_db(db_file):
 def db():
     with tempfile.TemporaryDirectory() as tmpdirname:
         yield init_db("{}/vulnerability.db".format(tmpdirname))
+
+
+@pytest.fixture(scope="session", autouse=True)
+def run_vmaas_mock():
+    """Run mocked VMaaS."""
+    vmaas_mock = subprocess.Popen(VMAAS_MOCK)
+    yield vmaas_mock
+    # teardown - stop vmaas_mock
+    vmaas_mock.terminate()
+
+
+@pytest.fixture(scope="session")
```

Comment:
How long does it take to setup full PostgreSQL server? I understand it would be difficult to mock the server as the code directly depends on PostgreSQL, but if it affects runtime of the tests by more than few seconds (current tests run for approx 1.3 sec), I would introduce a marker for slow tests (e.g. `slow`). This way the slow tests (anything that touches real database, network, ...) can run on travis when testing PRs etc., but are excluded by default when pytest is launched. We want developers to run tests very often during development and every second counts.

Reply:
It takes only 2 seconds to setup postgresql